src/client.rs

```diff
@@ -49,11 +49,13 @@ use tokio::time::sleep;
 use crate::errors::{Error, Result};
 use crate::frame::{Frame, FrameOutput, FramedMessage};
 use crate::ADDRESS_SEP;
+use flume::Receiver;
+use flume::Sender;
```

Comment:
Let's place it all on one line to be consistent with the `use` statements above.
```suggestion
use flume::{Receiver, Sender};
```